bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/api/dto/APIdata.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sunsynk.internal.api.dto;
+
+import java.time.Instant;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link APIdata} is the internal class for a Sunsynk Connect
+ * Account.
+ * 
+ * @author Lee Charlton - Initial contribution
+ */
+
+@NonNullByDefault
+public class APIdata {
+    public static String static_access_token = "";
+    private String access_token = "";
```

Comment:
Please check SAT for these kind of name warnings, you can fix them like this with annotations to map between java and json names
```suggestion
    @SerializedName("access_token")
    private String accessToken = "";
```

Reply:
Done